themes/tiddlywiki/vanilla/base.tid

```diff
@@ -938,6 +938,10 @@ canvas.tc-edit-bitmapeditor  {
 	display: block;
 }
 
+.tc-tiddler-body {
+	clear: both;
```

Comment:
hmmm, imo should be done with a real float clearfix assigned to the parent element. Otherwise we will suffer from 
really strange behaviour in different browsers. 

one possibility would be: 

```
.clearfix { display:block;} 

.clearfix:after {
     visibility: hidden;
     display: block;
     font-size: 0;
     content: " ";
     clear: both;
     height: 0;
     }
```